src/main/kotlin/com/jaredsburrows/license/internal/LicenseHelper.kt

```diff
@@ -80,6 +80,7 @@ object LicenseHelper {
     "MIT License" to "mit.txt",
     "http://opensource.org/licenses/MIT" to "mit.txt",
     "https://opensource.org/licenses/MIT" to "mit.txt",
+    "http://www.opensource.org/licenses/mit-license.php" to "mit.txt",
```

Comment:
Should this be in a separate PR?

Reply:
If you'd prefer. I'll back it out for now.